backend/postgres/schema.sql

```diff
@@ -0,0 +1,57 @@
+CREATE SCHEMA IF NOT EXISTS durabletask;
+
+CREATE TABLE IF NOT EXISTS durabletask.Instances (
+    InstanceID VARCHAR PRIMARY KEY NOT NULL,
+    ExecutionID VARCHAR NOT NULL,
+    Name VARCHAR NOT NULL, -- the type name of the orchestration or entity
+    Version VARCHAR NULL, -- the version of the orchestration (optional)
+    RuntimeStatus VARCHAR NOT NULL,
+    CreatedTime TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    LastUpdatedTime TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    CompletedTime TIMESTAMP NULL,
+    LockedBy VARCHAR NULL,
+    LockExpiration TIMESTAMP NULL,
+    Input VARCHAR NULL,
+    Output VARCHAR NULL,
+    CustomStatus VARCHAR NULL,
+    FailureDetails JSON NULL,
```

Comment:
Are you planning on querying the fields inside the FailureDetails section? If not, best to store JSON in a VARCHAR (i.e. TEXT) column. The JSON column (and in theory, JSONB should be used instead) is useful to be able to query inside a JSON document, but if it's just for storing data, using TEXT/VARCHAR is more efficient.